src/reconcile/rec_write.c

```diff
@@ -2968,6 +2977,8 @@ done:	if (F_ISSET(r, WT_REC_LOOKASIDE)) {
 		WT_ASSERT(session, r->unstable_txn != WT_TXN_NONE);
 		multi->page_las.max_timestamp = r->max_timestamp;
 		multi->page_las.unstable_timestamp = r->unstable_timestamp;
+		multi->page_las.unstable_durable_timestamp =
+		    r->unstable_durable_timestamp;
```

Comment:
Since the more general assertion didn't turn out to be OK - could you add an assertion in here that:

`r->unstable_durable_timestamps > r->unstable_timestamp`